rest-api-spec/src/main/resources/rest-api-spec/api/indices.stats.json

```diff
@@ -52,6 +52,11 @@
         "types" : {
           "type" : "list",
           "description" : "A comma-separated list of document types for the `indexing` index metric"
+        },
+        "include_segment_file_sizes": {
+          "type": "boolean",
+          "description": "Whether report the aggregated disk usage of each one of the Lucene index files.",
```

Comment:
Can you fix the grammar here, remove the period, and add a remark that it only applies if segments stats are requested?